examples/http/aws_rds_instance_info.js

```diff
@@ -0,0 +1,263 @@
+/**
+ * The driver gets AWS RDS instance metrics and uses the script item to make HTTP requests to the CloudWatch API.
+ * Communication protocol is https
+ */
+
+//These functions are used to compute hash-based message authentication codes (HMAC) using a specified algorithm.
+function sha256(message) {
+    return D.crypto.hash(message, "sha256", null, "hex");
+}
+
+function hmac(algo, key, message) {
+    key = D._unsafe.buffer.from(key);
+    return D.crypto.hmac(message, key, algo, "hex");
+}
+
+var region = "ADD_REGION";
+var secretKey = "ADD_SECRET_ACCESS_KEY";
```

Comment:
accessKey == username
secretKey == password 

You may add comments to use them as such and utilize D.device.username() as well as  D.device.password()